src/docfx/build/xref/XrefMap.cs

```diff
@@ -300,8 +301,19 @@ private IEnumerable<XrefSpec> ExpandInternalXrefSpecs()
             return loadedInternalSpecs;
         }
 
-        private (InternalXrefSpec spec, Document referencedFile) GetLatestInternalXrefMap(List<(InternalXrefSpec spec, Document referencedFile)> specs)
-            => specs.OrderByDescending(item => item.spec.Monikers.FirstOrDefault(), _context.MonikerProvider.Comparer).FirstOrDefault();
+        private (InternalXrefSpec spec, Document referencedFile) GetLatestInternalXrefMap(List<(InternalXrefSpec spec, Document referencedFile)> specs, List<string> monikers = null)
+        {
+            var orderedSpecs = specs.OrderByDescending(x => x.spec.Monikers.FirstOrDefault(), _context.MonikerProvider.Comparer);
```

Comment:
can these be pre-ordered?

Reply:
Would there be any difference with pre-ordered? We just sort them once, and the list is fairly small

Reply:
you are right, each resolve would trigger one sort